server/fishtest/util.py

```diff
@@ -33,6 +33,8 @@ def get_worker_key(task):
     suffix = UUID_MAP[username][uuid]
     if suffix != 0:
         worker_key += "-" + str(suffix)
+    if len(uuid) != 0:
```

Comment:
Is len(uuid) ever 0? Can this condition be removed?

Reply:
`uuid=""` for a very old test that lacks `"unique_key"`:
https://github.com/glinscott/fishtest/blob/55f4f9f66d618c6f627aec4222acb116cecb3b27/server/fishtest/util.py#L26

See this test on DEV:
https://dfts-0.pigazzini.it/tests/view/5134df478f0c3e3c12c7465c

Reply:
Ah, clear.